lib/src/connection.rs

```diff
@@ -298,6 +311,13 @@ impl ConnectionInfo {
                 ));
                 Encryption::Tls
             }
+            "bolt+ssc" => {
+                log::warn!(concat!(
+                    "This driver does not yet implement client-side routing. ",
+                    "It is possible that operations against a cluster (such as Aura) will fail."
+                ));
```

Comment:
We don't need the warning for `bolt` since that one is a direct connection and no routing is expected to happen.

Reply:
🤦